### PR TITLE
Setting Google Play package name for intent

### PR DESCRIPTION
**Pull request by Bersh**
I've added setting Google Play package name for rate intent (if available). This will allow to avoid the 'Complete Action by using (Play, Chrome, Web Browser)' dialog

### CODE REVIEW

File: library/src/main/java/hotchemi/android/rate/UriHelper.java [hunk 1]
```diff
@@ -13,4 +18,14 @@ static Uri getGooglePlay(String packageName) {
         return packageName == null ? null : Uri.parse(GOOGLE_PLAY + packageName);
     }
 
+    static boolean isPackageExists(Context context, String targetPackage) {
+        List<ApplicationInfo> packages;
```
Comment (hotchemi) [inline library/src/main/java/hotchemi/android/rate/UriHelper.java]: Please fix as following code.

```java
PackageManager pm = context.getPackageManager();
List<ApplicationInfo> packages = pm.getInstalledApplications(0);
```

And please tell me what `getInstalledApplications(0)` means(flag)?

**Comment by hotchemi** [comment]
Sorry for the delay.
I reviewed your contribution:)

**Comment by Bersh** [comment]
Hi @hotchemi !
Also sorry for the delay. I have fixed that you asked for.

`getInstalledApplications(0)` returns full list of installed application packages.

**Comment by hotchemi** [comment]
Thx! Perfect!

**Pull request merged**